### DISCUSSION TITLE
running CMake with CUDA does not produce "main.exe"

### DISCUSSION THREAD

**Discussion by gjnave** [ggml-org/llama.cpp#8126] [Q&A]
Curiously, the build is going through without errer( i ran it both in elevated command and using w64devkit) but at the end there is no: main.exe

Any ideas what is happening?

**Comment by AnonymousVibrate** [comment]
Same problem, it was renamed to `llama-server.exe` Using w64devkit  `make` produce .exe files. Using cmake does not produce the *.exe files. why is that ?

```
cmake -B build -DLLAMA_CUDA=ON
cmake --build build --config Release
```
Edit:
Solve it, the .exe files are within the **_llama.cpp/build/bin/Release_**

**Reply by gjnave** [reply]
ok .. you are correct. and totally name changed. llama-cli and llama-llava-cli
